src/pages/security/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import Hero from './_hero'
-import { Rewards, AboutDeriv, Scope, SubmitABugReport } from './_lazy-load'
+import { Rewards, Scope, SubmitABugReport } from './_lazy-load'
```

Comment:
I think we should also remove `Scope` from here as it also seems to be in the initial viewport for desktop.